charts/pulsar/values.yaml

```diff
@@ -281,7 +281,7 @@ zookeeper:
   # so the metrics are correctly rendered in grafana dashboard
   component: zookeeper
   # the number of zookeeper servers to run. it should be an odd number larger than or equal to 3.
-  replicaCount: 1
+  replicaCount: 3
```

Comment:
this is a huge breaking change, I don't think we should do it if there's no strong reason
I don't know which comments you're referring to, can you point them?

Reply:
Line #283 just above the changed line. It states >=3

Zookeeper with 1 replica seems like a bad default either way...

Reply:
We already provide an example for a minikube deployment https://github.com/apache/pulsar-helm-chart/blob/master/examples/values-minikube.yaml. That configuration overrides this value.

I agree that 3 makes more sense here, and I agree this is essentially a "breaking" change. I would prefer to merge this and go to 3.1.0 and call it out in the release notes.